eng/InstallRuntimes.proj

```diff
@@ -7,6 +7,7 @@
 
     $(MicrosoftDotnetSdkInternalVersion) - .NET SDK to use for testing
     $(InternalReleaseTesting) - if true, internal service release testing
+    $(PrivateBuildTesting) - if true, test against a locally built runtime 
```

Comment:
Not sure if `PrivateBuildTesting` does anything other than guard the config cleanup?

Reply:
No, it is passed through the debugger.tests.versions.txt to the SOS configs that use it to not run the single-file tests